Error during ./install.sh - Make logs

See below for make logs:
```
DKMS make.log for acpi_ec-v1.0.2 for kernel 6.4.3-hardened1-1-hardened (x86_64)
Wed Jul 19 17:46:46 BST 2023
  CC [M]  /var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.o
In file included from ./include/linux/linkage.h:7,
                 from ./include/linux/preempt.h:10,
                 from ./include/linux/spinlock.h:56,
                 from ./include/linux/mmzone.h:8,
                 from ./include/linux/gfp.h:7,
                 from ./include/linux/slab.h:15,
                 from ./include/linux/resource_ext.h:11,
                 from ./include/linux/acpi.h:13,
                 from /var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.c:13:
/var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.c: In function 'acpi_ec_create_dev':
./include/linux/export.h:27:22: error: passing argument 1 of 'class_create' from incompatible pointer type [-Werror=incompatible-pointer-types]
   27 | #define THIS_MODULE (&__this_module)
      |                     ~^~~~~~~~~~~~~~~
      |                      |
      |                      struct module *
/var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.c:115:39: note: in expansion of macro 'THIS_MODULE'
  115 |   if (IS_ERR(dev_class = class_create(THIS_MODULE, "chardev"))) {
      |                                       ^~~~~~~~~~~
In file included from ./include/linux/device.h:31,
                 from ./include/linux/acpi.h:14:
./include/linux/device/class.h:230:54: note: expected 'const char *' but argument is of type 'struct module *'
  230 | struct class * __must_check class_create(const char *name);
      |                                          ~~~~~~~~~~~~^~~~
/var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.c:115:26: error: too many arguments to function 'class_create'
  115 |   if (IS_ERR(dev_class = class_create(THIS_MODULE, "chardev"))) {
      |                          ^~~~~~~~~~~~
./include/linux/device/class.h:230:29: note: declared here
  230 | struct class * __must_check class_create(const char *name);
      |                             ^~~~~~~~~~~~
cc1: some warnings being treated as errors
make[1]: *** [scripts/Makefile.build:252: /var/lib/dkms/acpi_ec/v1.0.2/build/src/acpi_ec.o] Error 1
make: *** [Makefile:2026: /var/lib/dkms/acpi_ec/v1.0.2/build/src] Error 2
```

--- a/src/acpi_ec.c
+++ b/src/acpi_ec.c
@@ -17,6 +17,7 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/uaccess.h>
+#include <linux/version.h>
 
 MODULE_AUTHOR("Thomas Renninger <trenn@suse.de>");
 MODULE_DESCRIPTION("ACPI EC access driver");

--- a/src/acpi_ec.c
+++ b/src/acpi_ec.c
@@ -112,7 +112,12 @@ static int acpi_ec_create_dev(void) {
     return err;
   }
 
-  if (IS_ERR(dev_class = class_create(THIS_MODULE, "chardev"))) {
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 4, 0)
+  if (IS_ERR(dev_class = class_create(THIS_MODULE, "chardev")))
+#else
+  if (IS_ERR(dev_class = class_create("chardev")))
+#endif
+  {
     printk(KERN_ERR "acpi_ec: Failed to create a class\n");
     err = -1;
     goto error;

